src/Common/src/Interop/Unix/libgss/Interop.GssApi.cs

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static class GssApi
```

Comment:
The `libssl` directory is the only directory still named for the ultimate backing library.  It probably just got missed in the cleanup, and shouldn't be used as an example.